modules/persistent-storage-csi-secrets-store-driver-uninstall.adoc

```diff
@@ -23,16 +23,16 @@ To uninstall the Secrets Store CSI Driver Operator:
 .. On the *Instances* tab, for *secrets-store.csi.k8s.io*, on the far left side, click the drop-down menu, and then click *Delete ClusterCSIDriver*.
 .. When prompted, click *Delete*.
 . Verify that the CSI driver pods are no longer running.
-. Uninstall the Secrets Store CSI Driver Operator:
+. Uninstall the {secrets-store-operator}:
 +
 [NOTE]
 ====
 Before you can uninstall the Operator, you must remove the CSI driver first.
 ====
 +
 .. Click *Operators* → *Installed Operators*.
-.. On the *Installed Operators* page, scroll or type Secrets Store CSI into the *Search by name* box to find the Operator, and then click it.
+.. On the *Installed Operators* page, scroll or type "Secrets Store CSI" into the *Search by name* box to find the Operator, and then click it.
```

Comment:
```suggestion
.. On the *Installed Operators* page, scroll or type `Secrets Store CSI` into the *Search by name* box to find the Operator, and then click it.
```
Backticks instead of quotation marks? https://redhat-documentation.github.io/supplementary-style-guide/#text-entry

Reply:
@lpettyjo Same here, okay with this change?

Reply:
@bergerhoffer I think that quotes are correct here because you are indicating a text string to type in the box.

Reply:
Will leave as is